### PR TITLE
Adding the AntDeploy help to deploy applications

### PR DESCRIPTION
**Pull request by yuzd**
Adding the AntDeploy help to deploy applications.

AntDeploy - Visual stutio plugin to deploy applications to remote server(iis,windowsService,docker) support netframwork and dotnetcore,support rollback and increment deploy

### CODE REVIEW

**Comment by yuzd** [comment]
* [AntDeploy](https://github.com/yuzd/AntDeploy) - Visual stutio plugin to deploy applications to remote server(iis,windowsService,docker) support netframwork and dotnetcore,support rollback and increment deploy

**Comment by zihotki** [comment]
Hey, it would be good to have more documentation translated to English. Currently I was able to get only a vague idea about what it does and no idea about why and how it compares to existing options like MS Deploy and existing docker/k8s cli tools.

**Comment by yuzd** [comment]
Thanks , I see

File: README.md [hunk 1]
```diff
@@ -267,7 +267,7 @@ To the extent possible under law, [Vitali Fokin](https://github.com/quozd) has w
 * [.NET Compiler Platform ("Roslyn") Analyzers](https://github.com/dotnet/roslyn-analyzers) - A number of Roslyn diagnostic analyzers initially developed to help flesh out the design and implementation of the static analysis APIs.
 * [NDepend](https://www.ndepend.com) - is a Visual Studio and VS Team Services extension that estimates your .NET code quality and Technical-Debt, that lets create code rules with C# LINQ syntax, visualize code structure and focus on changes and evolution. **[$]**
 * [App.Metrics](https://github.com/AppMetrics/AppMetrics) - App Metrics is an open-source and cross-platform .NET library used to record and report metrics within an application and reports it's health. See the [docs](https://www.app-metrics.io/) for me details.
-* [CodeMaid](http://www.codemaid.net/) - Visual studio extension to cleanup, dig through and simplify C#, C++, F#, VB, PHP, JSON, XAML, XML, ASP, HTML, CSS, LESS, SCSS, JavaScript and TypeScript coding.
+* [CodeMaid](http//www.codemaid.net/) - Visual studio extension to cleanup, dig through and simplify C#, C++, F#, VB, PHP, JSON, XAML, XML, ASP, HTML, CSS, LESS, SCSS, JavaScript and TypeScript coding.
```
Comment (MaceWindu) [inline README.md]: whoops

**Pull request closed without merging**